website/docs/d/concrete_device.html.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: "aci"
+page_title: "ACI: aci_concrete_device"
+sidebar_current: "docs-aci-data-source-concrete_device"
+description: |-
+  Data source for ACI Concrete Device
+---
+
+# aci_concrete_device #
+
+Data source for ACI Concrete Device
+
+## API Information ##
+
+* `Class` - vnsCDev
+* `Distinguished Name` - uni/tn-{tenant_name}/lDevVip-{device_name}/cDev-{concrete_device_name}
+
+## GUI Information ##
+
+* `Location` - Tenant -> Services -> Devices -> Concrete Device
+
+## Example Usage ##
+
+```hcl
+data "aci_concrete_device" "example" {
+  l4_l7_devices_dn  = aci_l4_l7_devices.example.id
+  name              = "example"
+}
+```
+
+## Argument Reference ##
+
+* `l4-l7_devices_dn` - (Required) Distinguished name of parent L4-L7 Devices object.
+* `name` - (Required) Name of object Concrete Device.
+
+## Attribute Reference ##
+* `id` - Attribute id set to the Dn of the Concrete Device.
+* `annotation` - (Optional) Annotation of object Concrete Device.
+* `name_alias` - (Optional) Name Alias of object Concrete Device.
+* `vcenter_name` - (Optional) The virtual center name on which the device is hosted in the L4-L7 device cluster. It uniquely identifies the virtual center. Type: String.
+* `vm_name` - (Optional) The virtual center VM name on which the device is hosted in the L4-L7 device cluster. It uniquely identifies the VM. Type: String.
+* `relation_vns_rs_c_dev_to_ctrlr_p` - (Optional) Represents the relation from a Concrete Device to a VMM Controller (class vmmCtrlrP). It is an implicit relation to validate the controller profile. Type: String.
```

Comment:
What does "It is an implicit relation to validate the controller profile." mean?

Reply:
Removed